docs/installation.rst

```diff
@@ -4,153 +4,126 @@
+* `Python 3`_ >= 3.2.2. Even though Python 3 is not the default Python version on
+  most systems, it is usually available as an optional install.
+* OpenSSL_ >= 1.0.0
+* libacl_ (that pulls in libattr_ also)
+* liblz4_
+* some Python dependencies, pip will automatically install them for you
+* optionally, the llfuse_ Python package is required if you wish to mount an
+  archive as a FUSE filesystem. FUSE >= 2.8.0 is required for llfuse.
 
-    # ACL support Headers + Library
-    apt-get install libacl1-dev libacl1
+In the following, the steps needed to install the dependencies are listed for a
+selection of platforms. If your distribution is not covered by these
+instructions, try to use your package manager to install the dependencies.  On
+FreeBSD, you may need to get a recent enough OpenSSL version from FreeBSD
+ports.
 
-    # lz4 super fast compression support Headers + Library
-    apt-get install liblz4-dev liblz4-1
+After you have installed the dependencies, you can proceed with steps outlined
+under :ref:`pip-installation`.
 
-    # if you do not have gcc / make / etc. yet
-    apt-get install build-essential
+Debian / Ubuntu
+~~~~~~~~~~~~~~~
 
-    # optional: FUSE support - to mount backup archives
-    # in case you get complaints about permission denied on /etc/fuse.conf:
-    # on ubuntu this means your user is not in the "fuse" group. just add
-    # yourself there, log out and log in again.
-    apt-get install libfuse-dev fuse pkg-config
+Install the dependencies with development headers::
 
-    # optional: for unit testing
-    apt-get install fakeroot
+    sudo apt-get install python3 python3-dev python3-pip python-virtualenv
+    sudo apt-get install libssl-dev openssl
+    sudo apt-get install libacl1-dev libacl1
+    sudo apt-get install liblz4-dev liblz4-1
+    sudo apt-get install build-essential
+    sudo apt-get install libfuse-dev fuse pkg-config    # optional, for FUSE support
 
+In case you get complaints about permission denied on ``/etc/fuse.conf``: on
+Ubuntu this means your user is not in the ``fuse`` group. Add yourself to that
+group, log out and log in again.
 
-Korora / Fedora 21 preparations (git/pypi)
-------------------------------------------
+Fedora / Korora
+~~~~~~~~~~~~~~~
 
-::
+Install the dependencies with development headers::
 
-    # Python 3.x (>= 3.2) + Headers, Py Package Installer, VirtualEnv
     sudo dnf install python3 python3-devel python3-pip python3-virtualenv
-
-    # we need OpenSSL + Headers for Crypto
     sudo dnf install openssl-devel openssl
-
-    # ACL support Headers + Library
     sudo dnf install libacl-devel libacl
-
-    # lz4 super fast compression support Headers + Library
     sudo dnf install lz4-devel
+    sudo dnf install fuse-devel fuse pkgconfig         # optional, for FUSE support
 
-    # optional: FUSE support - to mount backup archives
-    sudo dnf install fuse-devel fuse pkgconfig
-    
-    # optional: for unit testing
-    sudo dnf install fakeroot
```

Comment:
I only removed the one for fakeroot. It's only needed for a dev installation and I put a note in the git installation instructions that fakeroot (and Cython) are required for that.